frontend/rolecall/src/app/api/performance-api.service.ts

```diff
@@ -145,6 +147,8 @@ export class PerformanceApi {
             custom_groups: val.positions.map(pos => {
               return {
                 position_uuid: String(pos.positionId),
+                // YHE
+                position_order: pos.casts[0].members[0].order,
```

Comment:
Could you explain this one a bit since I'm not yet familiar with the data model. So the order will always depend only on the first of many "casts" and the first of many "members"?

Reply:
I tried to do this with the separate email.